### PR TITLE
Polish text in the "Scenes and Nodes" section

### PR DESCRIPTION
**Pull request by d9n**
None of the changes in this CL change the meaning or spirit of the
underlying text. Instead, my goal was to tidy up miscellaneous things,
like:

* Remove passive voice
* Tighten wording when possible
* Clean up misc grammar nits and other errors
* Relayout text to fill 80 columns in some cases

### CODE REVIEW

**Comment by d9n** [comment]
Hello, I'm just getting started with learning about Godot. I'm quite interested in giving it a run and experimenting with it.

I have some experience working with doc writers. Even though I'm not a professional myself, I thought I could make some proposals to help the text read a little bit better.

I do worry I'm accidentally stepping on toes. If you are OK with what I did here, I will continue to make more patches for later tutorials as I read through them. Just let me know!

File: learning/step_by_step/scenes_and_nodes.rst [hunk 1]
```diff
@@ -44,65 +44,65 @@ children. When arranged in this way, the nodes become a **tree**.
...
+but for the game to start, one of them must be selected as the main scene.
+
+Basically, the Godot editor is a **scene editor**. It has plenty of tools for
+editing 2D and 3D scenes as well as user interfaces, but the editor is based on
+the concept of editing a scene and the nodes that compose it.
 
 Creating a new project
 ----------------------
 
-Theory is boring, so let's change the subject and go practical. Following a
```
Comment (d9n) [inline learning/step_by_step/scenes_and_nodes.rst]: (I modified this text because theory isn't boring! :)

**Comment by akien-mga** [comment]
The changes look fine :)

> I have some experience working with doc writers. Even though I'm not a professional myself, I thought I could make some proposals to help the text read a little bit better.
> 
> I do worry I'm accidentally stepping on toes. If you are OK with what I did here, I will continue to make more patches for later tutorials as I read through them. Just let me know!

Your contributions will definitely be welcome, and you're not stepping on anyone's toes by proposing such changes, so please continue :)

**Comment by d9n** [comment]
Thank you very much. I'll keep the patches coming and look forward to further reviews!

**Pull request merged**